src/app/static/src/scss/partials/genericChart.scss

```diff
@@ -0,0 +1,23 @@
+.chart-container {
+  overflow-y: scroll;
+  width: 100%;
+}
+.chart {
+  width: 100%;
+}
+.annotation-text {
+  font-weight: bold;
+}
+
+.modebar-group {
+  a[data-title='Zoom'],
+  a[data-title='Pan'],
+  a[data-title='Box Select'],
+  a[data-title='Lasso Select'],
+  a[data-title='Zoom in'],
+  a[data-title='Zoom out'],
+  a[data-title='Autoscale'],
+  a[data-title='Reset axes'] {
+    display: none;
+  }
```

Comment:
There doesn't seem to be a better way than this to selectively hide all but the download image icon in the Plotly modebar..